app/models/recipe.rb

```diff
@@ -1,3 +1,13 @@
 class Recipe < ApplicationRecord
+  has_many :recipe_ingredients
+  has_many :ingredients, through: :recipe_ingredients
+
    validates_presence_of :name, :complexity, :genre
+
+   def total_cost
+    ingredients.sum do |ingredient|
```

Comment:
Nice work encapsulating this logic in a model method! We can use pure ActiveRecord here instead of Ruby though: `ingredients.sum(:cost)`  --> No need for a do block

Reply:
When you add a method to your model, don't forget to test it!